src/AccessibilityInsights.SharedUx/Controls/SettingsTabs/ApplicationSettingsControl.xaml

```diff
@@ -210,8 +210,16 @@
                     <ColumnDefinition Width="151"/>
                     <ColumnDefinition Width="50"/>
                 </Grid.ColumnDefinitions>
-                <Label Name="lblPlayScanningSoundLabel" Grid.Column="0" Content="{x:Static Properties:Resources.lblPlayScanningSoundLabelContent}" Padding="0" Style="{StaticResource LblText}" VerticalAlignment="Center"/>
-                <ToggleButton Grid.Column="2" Style="{StaticResource tgbSlider}" Name="tgbtnPlayScanningSound" AutomationProperties.LabeledBy="{Binding ElementName=lblPlayScanningSoundLabel}" IsChecked="{Binding PlayScanningSound}"/>
+                <Label Name="lblSoundFeedbackLabel" Grid.Column="0" Content="{x:Static Properties:Resources.lblSoundFeedbackLabelContent}" Padding="0" Style="{StaticResource LblText}" VerticalAlignment="Center"/>
+                <ComboBox Grid.Column="2" x:Name ="soundFeedbackComboBox"
+                          SelectedIndex="{Binding SoundFeedbackMode}"
+                          AutomationProperties.Name="{x:Static Properties:Resources.ApplicationSettingsControl_SoundFeedbackModeComboboxAutomationName}"
+                          PreviewKeyDown ="cbComboBox_PreviewKeyDown"
+                          FocusVisualStyle="{DynamicResource {x:Static SystemParameters.FocusVisualStyleKey}}">
+                    <ComboBoxItem x:Name="cbiSFAUTO" Content="{x:Static Properties:Resources.cbiSFAUTOAutomationPropertiesName}" AutomationProperties.Name="{x:Static Properties:Resources.cbiSFAUTOAutomationPropertiesName}"/>
```

Comment:
naming nit: Having AUTO, ALWAYS, and NEVER in all caps is counterintuitive. Can we use Auto, Always, and Never, instead? This would apply to both the item name and the string resource name. 

The all caps in the highlight combo comes from the fact that they're abbreviations (HLTT = HighLighter + ToolTop), but that doesn't apply here.